lib/fluent/plugin/in_systemd.rb

```diff
@@ -31,6 +32,7 @@ def start
     end
 
     def shutdown
+      super
       @running = false
       @thread.join
       pos_writer.shutdown
```

Comment:
could we call super at the end of shutdown or the pos_writer will never get shutdown.